docker/Dockerfile.xenial_debug

```diff
@@ -19,6 +22,7 @@ RUN apt-get update && \
       elvis-tiny \
       lsof \
       linux-tools-generic \
-      sudo &&  apt-get upgrade -y && \
-    rm -rf /var/lib/apt/lists/*
-
+      sudo
+RUN apt-get upgrade -y
+RUN apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /var/log/*log /var/log/apt/* /var/lib/dpkg/*-old /var/cache/debconf/*-old
```

Comment:
this looks like some decent cleanup.  can you rework this PR just with this change?

Reply:
and also the ENV frontend.